db/migrate/20160310033614_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
+      t.text :contents
+      t.references :post, index: true, foreign_key: true
```

Comment:
Oh, this is nice. We should use this for the other "foreign keys". Also is it possible to but "NOT NULL" on these?